data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -ex
+
+IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic:master"}
+IRONIC_INSPECTOR_IMAGE=${IRONIC_INSPECTOR_IMAGE:-"quay.io/metal3-io/ironic-inspector:master"}
+IPA_DOWNLOADER_IMAGE=${IPA_DOWNLOADER_IMAGE:-"quay.io/metal3-io/ironic-ipa-downloader:master"}
+COREOS_DOWNLOADER_IMAGE=${COREOS_DOWNLOADER_IMAGE:-"quay.io/openshift-metal3/rhcos-downloader:master"}
+
+# This image is templated in via the installer pkg/asset/ignition/bootstrap/bootstrap.go
+RHCOS_IMAGE_BASE_URL="{{.BootImage}}"
+
+# First we stop any previously started containers, because ExecStop only runs when the ExecStart process
+# e.g this script is still running, but we exit if *any* of the containers exits unexpectedly
+for name in ironic-api ironic-conductor ironic-inspector dnsmasq httpd mariadb ipa-downloader coreos-downloader; do
+    podman ps | grep -w "$name$" && podman kill $name
+    podman ps --all | grep -w "$name$" && podman rm $name -f
+done
+
+# Start the provisioning nic if not already started
+# FIXME we should detect the nic in the ironic container (currently relies on the ifname, and won't pick 
+# up the description added here)
+# FIXME The IP/subnet and gateway should be provided via survey input?
+# FIXME: the ironic containers currently bind to all interfaces, which is not secure in the
+# case of the cluster, but we'll need the installer to access the public nic on the boostrap node
+# so probably we'll need a forwarding rule after https://github.com/metal3-io/ironic-image/pull/56
```

Comment:
I'm not clear on what the fix is here - change ironic to only bind to the provisioning interface?  ensure the installer is only using that IP?

Reply:
Ah thanks for the reminder, yeah the fix is to bind only on the provisioning nic, but I couldn't do that previously because terraform was accessing ironic on the host via the baremetal bridge - I'll re-test with https://github.com/metal3-io/ironic-image/pull/56 locally as I think it should work now that this series switches terraform to reference the BootstrapProvisioningIP.

Reply:
I removed the FIXME comments for now, added a link to https://github.com/openshift/installer/issues/2091 and will follow-up wrt the bind IP question via update of the ironic image (which can only really happen after this initial integration lands I think).